templates/admin/src/components/theme/drawer/UncontrolledDrawer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { getGlobalInstance } from 'plume-ts-di';
+import { useOnDependenciesChange } from '../../../lib/react-hooks-alias/ReactHooksAlias';
+import useToggle from '../../../lib/react-hook-toggle/ReactHookToggle';
+import { UncontrolledDrawerTypeProps } from '../../../lib/plume-admin-theme/drawer/DrawerProps';
+import PlumeAdminTheme from '../../../lib/plume-admin-theme/PlumeAdminTheme';
+
+/**
+ * When a drawer is a all page, there is no state to control its opening / closing
```

Comment:
is a whole page* ?
pas compris pourquoi il y aurait pas de state ?

Reply:
y a pas de state dans le parent pour controler l'ouverture / la fermeture. Donc c'est lui même qui gère son état d'ouverture / fermeture et le parent ne se charge de rien

Reply:
Dans ce cas, le composant est mal nommé, c'est plutôt `fullPageDrawer` ou quelque chose dans le genre